Nightwatch.Resources/Shell.fs

```diff
@@ -10,9 +10,12 @@ open Nightwatch.Core.Resources
 
 let private create (processController : Process.Controller)
                    (param : IDictionary<string, string>) =
-    let command = param.["cmd"]
+    let commandParams = param.["cmd"].Split(' ')
```

Comment:
1\. Usually you should prefer [the overload with `StringSplitOptions`](https://docs.microsoft.com/en-us/dotnet/api/system.string.split?view=netstandard-2.0#System_String_Split_System_Char___System_StringSplitOptions_) (and pass `RemoveEmptyEntries`). Otherwise samples with multiple spaces between arguments won't parse properly.

Reply:
2\. Currently your solution won't work if we need command with spaces. I suggest to add a separated parameter for arguments, like this:

```yaml
cmd: C:\Program Files\PowerShell\pwsh.exe
args: -File script.ps1 # will be split by space
```

Although that solution is still far from ideal. E.g. cases when we have more than one space inside of command (`-File "C:\Path  with  double  spaces  between words\script.ps1"` will be broken)¹.

Another, ideal, long-term variant would be to add an actual YAML array of strings as a parameters:

```yaml
cmd: C:\Program Files\PowerShell\pwsh.exe
args:
  - '-File'
  - '"C:\Program Files\Windows PowerShell  with  spaces\"'
```

and not parse it at all.

_(Unfortunately, we have some problems with .NET process API, so we will still have to merge the arguments before passing them to the operating system. but that's a separate issue.)_

¹ GitHub markdown engine have removed the double spaces for some reason, but they should be there.